phoenix-core/src/main/java/org/apache/phoenix/util/UpgradeUtil.java

```diff
@@ -1400,6 +1393,43 @@ public static void syncUpdateCacheFreqAllIndexes(PhoenixConnection conn, PTable
         }
     }
 
+    private static void iterateOverChildViewAndSyncCacheFreq(
```

Comment:
Rename this method since we aren't iterating over anything. Maybe after combining this with `getViewAndSyncCacheFreqForIndexes()` we can just call it the latter.